test/e2e/cht-docker-utils.js

```diff
@@ -14,7 +14,20 @@ const dockerHelperScript = path.resolve(dockerHelperDirectory, './cht-docker-com
 const downloadDockerHelperScript = () => new Promise((resolve, reject) => {
   const file = fs.createWriteStream(dockerHelperScript, { mode: 0o755 });
   https
-    .get('https://raw.githubusercontent.com/medic/cht-core/master/scripts/docker-helper-4.x/cht-docker-compose.sh', (response) => {
+    .get('https://raw.githubusercontent.com/medic/cht-core/master/scripts/docker-helper/cht-docker-compose.sh', (response) => {
```

Comment:
I was not aware we were using docker-helper in cht-conf. Feels a bit fragile, especially because we're targeting the file from `master`. I don't like it :D 

Thanks for catching this!